### ISSUE TITLE
Nonce is not taken in Authorization header of REGISTER

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#43]
```
http://code.google.com/p/doubango/issues/detail?id=19
```

Original issue reported on code.google.com by `boss...@yahoo.fr` on 25 Aug 2010 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I have the issue on some Alcatel-Lucent servers.
```

Original comment by `boss...@yahoo.fr` on 25 Aug 2010 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.279.
```

Original comment by `boss...@yahoo.fr` on 7 Sep 2010 at 8:11
- Changed state: **Fixed**
